denso_robot.py

```diff
@@ -100,3 +110,8 @@ def check_sample_mounted(self, mount, puck_pos, pin_pos):  # is the correct samp
                     return UNMOUNT_FAILURE
         else:
             return MOUNT_STEP_SUCCESSFUL  # always successful if robot is not online
+
+# This is a generic interface to give robots access to the robot_lib api
+class OphydRobot(DensoRobot):
```

Comment:
I'm not sure I understand what you mean to do here - you pass in a robot object, but also subclass DensoRobot?